.github/workflows/workflow.yml

```diff
@@ -34,3 +31,21 @@ jobs:
       run: |
        sudo chmod -R a+rxw /home
        make test
+  clang-fips-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup Bazel
+      run: sudo build/install-bazel.sh
+    - name: Install dependencies
+      run: sudo apt-get update && sudo apt-get -y install make cmake ninja-build build-essential
+    - name: Setup clang
+      env:
+        CLANG_VERSION: '12.0.0'
```

Comment:
we can omit this env var right? `install-clang-ci.sh` currently hardcoded this as 12.0.0 as well.